src/content/en/2022/javascript.md

```diff
@@ -16,7 +16,1010 @@ featured_stat_2: TODO
+
+## How much JavaScript do we load?
+
+To begin, we'll assess the amount of JavaScript web developers ship on the web. After all, before improvements can be made, an assessment of the current landscape must be performed.
+
+{{ figure_markup(
+    image="bytes-per-page.png",
+    caption="Distribution of the amount of JavaScript loaded per page.",
+    description="TODO",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTok8UGqYaA58uKaASB2pfM2jlmQu6g3kwHxB6Lb8L7dbccyDtQht823YhchdXRG8SZpB3asRayJI97/pubchart?oid=807278764&format=interactive",
+    sheets_gid="1117050794",
+    sql_file="bytes_2022.sql"
+  )
+}}
+
+As was the case last year, this year marks yet another increase in the amount of JavaScript shipped to browsers. [From 2021](../2021/javascript#how-much-javascript-do-we-load) to 2022, an increase of 8% for mobile devices was observed, whereas desktop devices saw an increase of 10%. While this increase is less steep than in previous years, it's nonetheless the continuation of a concerning trend. While device capabilities continue to improve, not every one is running the latest device. The fact remains that more JavaScript equates to more strain on a device's resources.
+
+{{ figure_markup(
+    image="unused-js.png",
+    caption="Distribution of the amount of unused JavaScript bytes.",
+    description="TODO",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTok8UGqYaA58uKaASB2pfM2jlmQu6g3kwHxB6Lb8L7dbccyDtQht823YhchdXRG8SZpB3asRayJI97/pubchart?oid=1640758017&format=interactive",
+    sheets_gid="943216000",
+    sql_file="unused_js_bytes_distribution.sql"
+  )
+}}
+
+According to [Lighthouse](https://web.dev/unused-javascript/), the median mobile page loads 162 KB of unused JavaScript. At the 90th percentile, 604 KB of JavaScript are unused. This is a slight uptick from last year, where the median and 90th percentile of unused JavaScript was 155 KB and 598 KB, respectively. All of this represents a _very_ large amount of unused JavaScript, especially when you consider that this analysis tracks the _transfer size_ of JavaScript resources which, if compressed, means that the decompressed portion of used JavaScript may be a lot larger than the chart suggests.
+
+When contrasted with the total number of bytes loaded for mobile pages at the median, unused JavaScript accounts for 35% of all loaded scripts. This is down slightly from last year's figure of 36%, but is still a significantly large chunk of bytes loaded that go unused. This suggests that many pages are loading scripts that may not be used on the current page, or are triggered by interactions later on in the page lifecycle, and may benefit from [dynamic `import()`](#dynamic-import) to reduce startup costs.
+
+## JavaScript requests per page
+
+Every resource on a page accounts for one request, and scripts are no different. When many scripts are loaded, it's reasonable to think that more bytes of JavaScript may be loaded as a result.
```

Comment:
```suggestion
When many scripts are loaded on a page, there's a higher potential more bytes of JavaScript may be loaded as a result.
```

Reply:
Thoughts on this wording @tunetheweb?

Reply:
To be honest I think it's kind of redundant. "When you load more, you're probably loading more".

I do think the potential for clashes is higher and maybe that's a more interesting take?